package.json

```diff
@@ -14,8 +14,8 @@
     "write-heading-ids": "docusaurus write-heading-ids"
   },
   "dependencies": {
-    "@docusaurus/core": "^2.2.0",
-    "@docusaurus/preset-classic": "^2.2.0",
+    "@docusaurus/core": "^2.3.1",
+    "@docusaurus/preset-classic": "^2.3.1",
     "@docusaurus/theme-search-algolia": "^2.2.0",
```

Comment:
I think this needs to be bumped as well. The [docs](https://docusaurus.io/docs/installation#updating-your-docusaurus-version) mention 

> Note that all `@docusaurus/`-namespaced packages should be using the same version.

Reply:
Good catch, this should be fixed now